File: src/main/java/textGame/GameObject.java
package textGame;

public class GameObject {
    private String name;
    private String description;
    public String getName() {
        return this.name;
    }
    public String getDescription() {
        return this.description;
    }
    public void setName(String name) {
        this.name = name.replace(" ", "_");
    }
    public void setDescription(String description) {
        this.description = description;
    }
    public String toString() {
        return this.name;
    }
}

File: src/main/java/textGame/GameObject.kt
package textGame

open class GameObject : Inspectable {
    private var name: String? = null
    private var description: String? = null
    override fun getName(): String {
        return name!!
    }

    fun setName(name: String) {
        this.name = name.replace(" ", "_")
    }
    fun getDescription(): String {
        return description!!
    }
    fun setDescription(description: String) {
        this.description = description
    }

    override fun toString(): String {
        return name!!
    }

    override fun getAttributes(): Array<String> {
        return arrayOf("Description:" + description)
    }
}
